res/skins/QMLDemo/Deck.qml

```diff
@@ -1,400 +1,353 @@
-                            target: syncButtonBgImage
-                            source: "../LateNight/palemoon/buttons/btn_embedded_sync_active.svg"
-                        }
-
-                    }
+                anchors.top: parent.top
+                anchors.bottom: parent.bottom
+                anchors.left: waveformBarVSeparator.left
+                anchors.leftMargin: rateSlider.width
+                width: 2
+                color: infoBar.lineColor
+            }
 
-                    background: Rectangle {
-                        anchors.fill: parent
+            InfoBarButton {
+                anchors.top: parent.top
+                anchors.bottom: parent.bottom
+                anchors.left: waveformBarHSeparator1.left
+                width: rateSlider.width
+                group: "[EffectRack1_EffectUnit2]"
+                key: "group_" + root.group + "_enable"
+                checkable: true
+                activeColor: Theme.deckActiveColor
+
+                foreground: Text {
+                    anchors.centerIn: parent
+                    text: "FX 2"
+                    verticalAlignment: Text.AlignVCenter
+                    horizontalAlignment: Text.AlignHCenter
+                    font.family: Theme.fontFamily
+                    font.bold: true
+                    font.pixelSize: Theme.textFontPixelSize
+                    color: infoBar.textColor
+                }
 
-                        data: Image {
-                            id: syncButtonBgImage
+            }
 
-                            anchors.fill: parent
-                        }
+            Rectangle {
+                id: waveformBarHSeparator2
 
-                    }
+                anchors.top: parent.top
+                anchors.bottom: parent.bottom
+                anchors.left: waveformBarHSeparator1.right
+                anchors.leftMargin: rateSlider.width
+                width: 2
+                color: infoBar.lineColor
+            }
 
+            Text {
+                id: waveformBarPosition
+
+                anchors.top: parent.top
+                anchors.bottom: parent.bottom
+                anchors.left: waveformBarHSeparator2.right
+                anchors.leftMargin: 5
+                verticalAlignment: Text.AlignVCenter
+                horizontalAlignment: Text.AlignLeft
+                font.family: Theme.fontFamily
+                font.pixelSize: Theme.textFontPixelSize
+                color: infoBar.textColor
+                text: {
+                    let positionSeconds = samplesControl.value / 2 / sampleRateControl.value * playPositionControl.value;
+                    if (isNaN(positionSeconds))
+                        return "";
+
+                    var minutes = Math.floor(positionSeconds / 60);
+                    var seconds = positionSeconds - (minutes * 60);
+                    var subSeconds = Math.trunc((seconds - Math.trunc(seconds)) * 100);
```

Comment:
It's centiseconds actually. Should I multiply by 100 instead?

Reply:
Sorry didn't pay close enough attention, should be centiseconds technically, but I don't really care, subseconds should be fine as well. I care more about why var was chosen instead of let?

Reply:
Muscle memory trained from our pre-2. 4 controller engine. I always used let before that, and when I started mapping my controller with Mixxx I stumbled over this. I'll change it.

Reply:
Also make sure to use const when you can (not in this case since you're reassigning values but `positionSeconds` can be const).